packages/back-end/src/integrations/BigQuery.ts

```diff
@@ -122,4 +122,28 @@ export default class BigQuery extends SqlIntegration {
 
     return formatInformationSchema(results, "bigquery");
   }
+
+  async getTableData(
+    databaseName: string,
+    tableSchema: string,
+    tableName: string
+  ): Promise<{ tableData: null | unknown[]; refreshMS: number }> {
+    const sql = `SELECT
+          data_type,
+          column_name
+        FROM
+          ${databaseName}.${tableSchema}.INFORMATION_SCHEMA.COLUMNS
```

Comment:
We might want to quote this in backticks like we do for the other query since the databaseName or tableSchema might contain special characters.

Reply:
@jdorn Added the backticks to the from statement - I explored adding them to the `IN` statements a few lines below, but I wasn't able to get the query working as expected - if we need the backticks in the `FROM` statement, shouldn't we also add them the few lines below?

Reply:
The values in the where clause are already quoted with strings.